project/Dependencies.scala

```diff
@@ -20,7 +20,7 @@ object Dependencies {
 
   val Smithy = new {
     val org = "software.amazon.smithy"
-    val smithyVersion = "1.31.0"
```

Comment:
if we rename that to `smithyModelVersion` we'll probably get scala-steward to work

Reply:
how Steward works is a constant mystery to me

Reply:
yeah the algorithm is kinda funky :P